README.md

```diff
@@ -1,11 +1,74 @@
+[![Go Reference](https://pkg.go.dev/badge/github.com/ngrok/ngrok-go.svg)](https://pkg.go.dev/github.com/ngrok/ngrok-go)
+[![Go](https://github.com/ngrok/ngrok-go/actions/workflows/buildandtest.yml/badge.svg)](https://github.com/ngrok/ngrok-go/actions/workflows/buildandtest.yml)
 # ngrok-go
 
-**Note: This is alpha-quality software. Interfaces are subject to change
-without warning**
+ngrok is a globally distributed reverse proxy commonly used for quickly getting a public URL to a service running inside a private network, such as your laptop. The ngrok agent is usually deployed inside a private network and is used to communicate with the ngrok cloud service.
 
-This is the ngrok agent in library form, suitable for integrating directly into
-an application. See `example/http.go` for example usage, or the tests in
-`online_test.go`.
+This is the ngrok agent in library form, suitable for integrating directly into your Go application. This allows you to quickly build ngrok into your application with no separate process to manage. 
```

Comment:
total np, but "your Go application" -> "Go applications" feels ever so slightly better.

Forking and adding this to an existing go application is a totally cool use-case too.